static/js/popovers.js

```diff
@@ -8,10 +8,14 @@ var current_actions_popover_elem;
 var current_flatpickr_instance;
 var current_message_info_popover_elem;
 var current_mobile_message_buttons_popover_elem;
+var send_later_popover_elem;
 var userlist_placement = "right";
 
 var list_of_popovers = [];
 
+var APPROX_HEIGHT = 375;
+var APPROX_WIDTH = 255;
```

Comment:
These variables seem like they should be inside the functions for the `send_later` popover, or named `APPROX_SEND_LATER_HEIGHT`.

Reply:
That definitely makes more sense.